app/models/concerns/foreman_bootdisk/host_ext.rb

```diff
@@ -33,11 +33,17 @@ def bootdisk?
     managed? && bootdisk_build? && SETTINGS[:unattended]
   end
 
+  def bootdisk_downloadable?
+    ['ia32', 'x64'].include?(architecture.try(:bootfilename_efi))
```

Comment:
Okay. It seemed closely related to `architecture#name` so I thought to use it rather than writing a new predicate method.
I suppose `architecture#intel?` would be meaningful.

```ruby
def intel?
  /i.86|x86[_-]64/ =~ name
end
```